source/common/http/codec_helper.h

```diff
@@ -13,9 +13,10 @@ class StreamCallbackHelper {
     }
     ASSERT(high_watermark_callbacks_ > 0);
     --high_watermark_callbacks_;
-    // TODO(alyssawilk) see if we can make this safe for disconnects mid-loop
     for (StreamCallbacks* callbacks : callbacks_) {
-      callbacks->onBelowWriteBufferLowWatermark();
+      if (callbacks) {
+        callbacks->onBelowWriteBufferLowWatermark();
+      }
```

Comment:
Sure thing.  Thankfully it'll fail tests as well, but now there will be a helpful explanation why :-)